src/services/item/plugins/app/chatBot/test/fixtures.ts

```diff
@@ -0,0 +1,41 @@
+import { ChatCompletion } from 'openai/resources/chat';
+
+import { ChatBotMessage } from '@graasp/sdk';
+
+import * as OpenAICompletion from '../openAICompletion';
+
+export const DOCKER_MOCKED_RESPONSE =
+  'Docker is a platform for developing, shipping, and running applications in lightweight, isolated containers that provide consistency and portability across different environments.';
+export const DOCKER_MOCKED_BODY: ChatBotMessage[] = [
+  {
+    role: 'system',
+    content: 'You are a docker expert',
+  },
+  {
+    role: 'assistant',
+    content: 'Welcome ! How can I help you with Docker ?',
+  },
+  {
+    role: 'user',
+    content: 'What is Docker ?',
+  },
+];
+
+export const copyArray = (arr: Array<unknown>) => JSON.parse(JSON.stringify(arr));
+
+async function responseFactory(finish_reason: string | null, content?: string) {
+  return {
+    choices: [
+      {
+        finish_reason: finish_reason,
```

Comment:
I don't know if it's a specific variables otherwise please change to:
```suggestion
        finishReason: finishReason,
```

Reply:
The property is called `finish_reason` in the openAI types wo we can't do anything about it, but on our side we will try to stick to our conventions.